main.css

```diff
@@ -1,5 +1,5 @@
 .head {
-  background: #fc0;
+  background: red;
```

Comment:
Es mejor ponerlo en amarillo
```suggestion
  background: #fc0;
```

Reply:
De acuerdo lo cambiamos a amarillo, corrijo el código. Gracias
```suggestion
  background: #fc0;
```